### DISCUSSION TITLE
locales where "code" != $1

### DISCUSSION THREAD

**Discussion by Serikxx** [requarks/wiki#5016] [General]
Hi. Went to the wiki address. I clicked on install and an error came out: `delete from "locales" where "code" != $1 - update or delete on table "locales" violates foreign key constraint "users_localecode_foreign" on table "users"`. 
I wanted to clarify how to specify the correct value for locales or there inside. 
Postgresql output:
`select * from locales;
code |`

**Comment by arkansas-coder** [comment]
I'm having the same problem. Have you been able to solve it?
